__tests__/tree.test.js

```diff
@@ -0,0 +1,69 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const Tree = require('../lib/models/Tree');
+
+describe('Tree CRUD routes', () => {
+    beforeEach(() => {
+        return setup(pool);
+    });
+    //post 
+    it('adds a new tree', () => {
+        return request(app)
+            .post('/api/v1/trees')
+            .send({"treeName": "spruce", "treeType": "evergreen", "treeQuantity": "2"})
+            .then((res) => {
+                expect(res.body).toEqual([{
+                    treeId: '1',
+                    treeName: 'spruce',
+                    treeType: 'evergreen',
+                    treeQuantity: 2
+                }]);
+            });
+    });
+    //get 
+    it('returns all trees', () => {
+        Tree.insert({"treeName": "birch", "treeType": "dedeciduous hardwood", "treeQuantity": "1"})
```

Comment:
This works! You can also add a `beforeEach` on line 10.5 that inserts a new tree beforeEach test, that way you won't need to insert a new tree inside each `it()` :)